DataConnectors/NGINX/Connector_NGINX_agent.json

```diff
@@ -0,0 +1,149 @@
+{
+    "id": "NGINX HTTP Server",
+    "title": "NGINX HTTP Server",
+    "publisher": "Nginx",
+    "descriptionMarkdown": "NGINX HTTP Server data connector provides the capability to ingest [NGINX HTTP Server](https://nginx.org/en/) events into Azure Sentinel. Refer to [NGINX Logs documentation](https://nginx.org/en/docs/http/ngx_http_log_module.html) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **NGINXHTTPServer** in queries and workbooks. [Follow steps to get this Kusto Function>](https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/NGINX/NGINXHTTPServer.txt)",
```

Comment:
Please replace parser link with this link : https://aka.ms/sentinel-NGINXHTTP-parser

Reply:
updated